### PR TITLE
Update `faas-cli new --list` command in lab3

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Update `faas-cli new --list` command in lab3.

- Update list of available language templates
- Remove the paragraph at the end
```
Or alternatively create a folder containing a Dockerfile, then pick
the "Dockerfile" lang type in your YAML file.
```

### CODE REVIEW

File: lab3.md [hunk 1]
```diff
@@ -34,19 +34,22 @@ After that, to find out which languages are available type in:
...
 - node-arm64
 - node-armhf
+- php7
 - python
 - python-armhf
 - python3
+- python3-armhf
 - ruby
-
-Or alternatively create a folder containing a Dockerfile, then pick
```
Comment (alexellis) [inline lab3.md]: Can you change this to say that they can use the Dockerfile template for something completely custom?

**Comment by alexellis** [comment]
One idea for you.

Thanks for the interest.

File: lab3.md (hunk 1 above)
Comment (jonatasbaldin) [reply lab3.md]: Should this comment be a statement in the workshop or actually returned by the `faas-cli new --list` command? 🤔

**Comment by jonatasbaldin** [comment]
Should this comment be a statement in the workshop or actually returned by the faas-cli new --list command? 🤔

**Comment by jonatasbaldin** [comment]
hey @alexellis, any comments on this? ✨

**Comment by rgee0** [comment]
It should be a statement in the workshop to help draw the reader to the opportunity that the Dockerfile template presents

**Comment by jonatasbaldin** [comment]
Reintroduced the phrase :)

**Pull request merged**